omnigibson/utils/usd_utils.py

```diff
@@ -514,11 +514,11 @@ def get_column_filters(cls):
 
     @classmethod
     def get_max_contact_data_count(cls):
-        # 2x per finger link, to be safe.
-        # 2 here is not the finger count, it's the number of items we will record contacts with, per finger.
+        # 4x per finger link, to be safe.
+        # 4 here is not the finger count, it's the number of items we will record contacts with, per finger.
         # e.g. it's N such that if the finger is touching more than N items at once, only the first N are recorded.
-        # This number should very rarely go above 2.
-        return len(cls.get_column_filters()[0]) * 2
+        # This number should very rarely go above 4.
```

Comment:
was this just an empirical update? or another reason?

Reply:
Yes, just empirical. This causes an issue when the gripper finger is in contact with multiple links and messes up with the AG logic.